include/wabt/interp/interp-inl.h

```diff
@@ -42,7 +42,30 @@ inline bool FuncType::classof(const ExternType* type) {
 }
 
 inline FuncType::FuncType(ValueTypes params, ValueTypes results)
-    : ExternType(ExternKind::Func), params(params), results(results), func_types(nullptr) {}
+    : ExternType(ExternKind::Func),
+      kind(FuncType::TypeKind::Func),
+      canonical_index(kInvalidIndex),
+      canonical_sub_index(kInvalidIndex),
+      is_final_sub_type(true),
+      recursive_start(0),
+      recursive_count(0),
```

Comment:
Is there some reason not just just put the defaults like `= 0` into the class definition itself?

Reply:
Done